server/models/auth/forgotPassword.js

```diff
@@ -20,6 +20,24 @@ function forgotPassword({ email }) {
     const html = emailMessage(otp, host, port)
     sendEmail(email, subject, html)
 
+    pool.query(
+      `SELECT verified FROM user WHERE email=?`,
+      [email],
+      (error, results) => {
+        if (error) {
+          return reject(error)
+        }
+        if (!results.length) {
+          return reject('No account exists with the given email!')
+        }
+        const { verified } = results[0]
+        if (verified !== 1) {
+          return reject(
+            'The account with this email is not verified yet, please verify your email account first!'
```

Comment:
if email exists and not verified, it needs to get verified first

Reply:
Fine